### DISCUSSION TITLE
Interface size problem

### DISCUSSION THREAD

**Discussion by Limyx826** [OpenTTD/OpenTTD#10505] [Features]
Playing on a screen of 1920 x 1080 with OpenTTD being windowed. I found that the default option is quite painful to look at. If I scale it to 2x then everything is too big. I think there should be an option of icons in between 1x and 2x scale like 1.5x. Let see the vanilla scaling in the screenshot below. 

![image](https://user-images.githubusercontent.com/38696851/221197617-233d5815-ddc6-4a97-8b0b-323631e34ff3.png)
This is 1x size.

![image](https://user-images.githubusercontent.com/38696851/221197657-e0e0adf0-5cdf-4a13-a37f-934c7d8f5e41.png)
This is the default size set by auto scale. 

![image](https://user-images.githubusercontent.com/38696851/221197771-06b0ba0d-58e3-4277-9282-a5a72953bf75.png)
This is 2x scale, as you can see the UI is so big that it obscures some elements. 

Before UI scaling is a thing I always use a NewGRF called OpenGFX big GUI. It offers 1.5x and 2x scale which I find 1.5x is best suitable for the screen resolution I have. Too bad that the NewGRF haven't been updated for quite some time and that leaves certain icons not being up to scale which otherwise I will be continue using it without brothering with the new UI scaling. 
The screenshots below show 1.5x scale of big GUI. 
![image](https://user-images.githubusercontent.com/38696851/221198595-fc60567a-c73d-40ae-bfd1-0cf0ed095a0a.png)
![image](https://user-images.githubusercontent.com/38696851/221198640-d2edbc5e-1af2-467f-b04e-203a86f50240.png)
![image](https://user-images.githubusercontent.com/38696851/221198771-234e1472-7d5b-4c2a-ad4a-18c8f493fd7f.png)

Now 2x scale of big GUI which is the same as vanilla 2x scale. 
![image](https://user-images.githubusercontent.com/38696851/221199269-a52dac80-135f-4daa-80c5-92c3c0b898ae.png)
![image](https://user-images.githubusercontent.com/38696851/221199329-15bd8e7c-affe-4ffa-8a76-53401ef26fb9.png)

**Comment by Limyx826** [comment]
~I open a issue request at #10506.~

**Reply by PeterN** [reply]
Don't do that unless you have a PR to offer.

**Comment by PeterN** [comment]
The 1.5x interface scale option is literally there in your screenshots.

Currently all sprites can only be scaled by powers of 2, which is why at non-power of 2 scales they are padded instead.

**Reply by Limyx826** [reply]
Maybe I should have mention it is the sprites should have 1.5x scale.

**Comment by Limyx826** [comment]
I have updated the original discussion with more clarity that there should be 1.5x icons.

**Comment by Ufiby** [comment]
Try using with U&GUI

**Reply by Limyx826** [reply]
Like your works, haven't try this yet. Will do so later. 

Nevertheless, I prefer to have those integrated into the base game itself rather than relying on NewGRF.
